Week1/homework/style.css

```diff
@@ -1 +1,84 @@
-/* add your styling here */
\ No newline at end of file
+*, *:before, *:after {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font: inherit;
+  vertical-align: baseline;
+  box-sizing: inherit;
+}
+
+body {
+  background-color: black;
+  padding: 2%;
+}
+
+h1 {
+  display: inline-flex;
+  font-size: 8em;
+  font-weight: bold;
+  justify-content: center;
+  width: 100%;
+  height: auto;
+  box-sizing: border-box;
+  color: rgb(151, 53, 53);
+  text-shadow: 5px 5px 9px rgba(212, 137, 119, 0.671);
+}
+
+h2 {
+  display: flex;
+  justify-content: center;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 1.7em;
+  margin: 0.5em 1em 0.5em 1em;
+  font-weight: bold;
+}
+
+p {
+  display: flex;
+  justify-content: center;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+p.gamePlatform {
+  font-size: 1.2em;
+  color: rgb(55, 55, 145);
+  font-weight: bold;
+  margin: 0.5em 0.5em 0.5em 0.5em;
+}
+
+p.gameDeveloper {
+  font-size: 1.2em;
+  font-weight: bold;
+  color: rgb(201, 52, 41);
+  margin: 0em 0.5em 0.1em 0.5em;
+}
+
+.game_container {
+  float: left;

```

Comment:
Instead of using `float` I'd suggest exploring `flex` and/or `grid`. They're better suited for this layout.

Reply:
I actually salvaged the CSS from another assignment, will fix it asap. I think I just focused on the JS for this one 😅